cmd/link/ocmrole/cmd.go

```diff
@@ -79,7 +86,15 @@ func run(cmd *cobra.Command, argv []string) (err error) {
 	orgAccount, _, err := ocmClient.GetCurrentOrganization()
 	if err != nil {
 		reporter.Errorf("Error getting organization account: %v", err)
+		return err
+	}
+
+	if args.organizationID != "" && orgAccount != args.organizationID {
```

Comment:
I don't understand why let the users pass in this parameter, if we default to the current org, and only allow setting the current org?

Reply:
This is because we need the organization admin and if the user is not an org admin we need to let org admins to run the command by passing the org id

Reply:
@zgalor it's basically extra validation to ensure that the admin being sent this command is logged in to the correct org.